### DISCUSSION TITLE
Rendering in Pytorch3d

### DISCUSSION THREAD

**Discussion by GiilDe** [Genesis-Embodied-AI/genesis-world#663] [Q&A]
Is it possible to render a scene with pytorch3d? I want to use differentiable rending and since it doesn't seem to be supported internally, I was thinking to render using pytorch3d which supports this. If it is possible to use pytorch3d, will it be possible to backpropagate the gradients from the video back to the actions?

**Comment by Kashu7100** [comment]
@zhouxian Do you have any idea?
